fs-extra/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for fs-extra
+// Type definitions for fs-extra v2.0.0
```

Comment:
I'm not sure if this is the right way to version this, but I added this to ensure we're not confusing `1.x` usage with `2.x` usage.